debian/phosh-config-hwcomposer.maintscript

```diff
@@ -0,0 +1,4 @@
+rm_conffile /etc/systemd/system/phosh.service.d/40-droid-wait.conf 12+git20230814235440.4f5b86f.trixie.production~
```

Comment:
Shouldn't the version here be 13? Latest version in prod is `13+git20231101214714.859982b.trixie.production`

```
rm_conffile /etc/systemd/system/phosh.service.d/40-droid-wait.conf 12+git20230814235440.4f5b86f.trixie.production~
```

`13~` should work as well

Reply:
ok, using `13~` now.